ap_api/resources/get-quotes.api.mdx

```diff
@@ -0,0 +1,47 @@
+---
+id: get-quotes
+title: "Fetches the quotes provided to client applications."
```

Comment:
I would change to "Fetch the quotes..." to align with the other sidebar titles. Either all should be "Fetch" or all should be "Fetches"